openshift-kube-apiserver/admission/authorization/restrictusers/groupcache_test.go

```diff
@@ -22,3 +22,7 @@ func (g fakeGroupCache) GroupsFor(user string) ([]*userv1.Group, error) {
 	}
 	return ret, nil
 }
+
```

Comment:
squash into commit which introduces this code.

Reply:
Done. Squashed with `UPSTREAM: <carry>: openshift-kube-apiserver: add kube-apiserver patches`.